package.json

```diff
@@ -19,25 +19,25 @@
     "build": "rimraf dist && tsc && copyfiles -u 1 'src/ui/**/*' dist/src/ && copyfiles -u 1 'generated/*' dist/src/generated",
     "generate": "graphql-codegen",
     "start": "ts-node test/dev-server.ts",
-    "test": "jest --preset=\"ts-jest\" --forceExit"
+    "test": "vitest run"
```

Comment:
You can run `yarn vitest` to run Vitest in watch mode when developing locally :mechanical_arm: